packages/@okta/vuepress-site/docs/reference/api/iam-roles/index.md

```diff
@@ -0,0 +1,3027 @@
+  "permissions": [
+    "okta.users.create",
+    "okta.users.read",
+    "okta.groups.read",
+    "okta.users.profile.manage"
+  ]
+}
+```
+
+### Get Role
+<ApiLifecycle access="beta" />
+
+<ApiOperation method="get" url="/api/v1/iam/roles/${roleIdOrLabel}" />
+
+Gets a custom Role by its ID or label
+
+#### Request parameters
+
+| Parameter     | Description                          | Param Type   | DataType                    | Required |
+| :------------ | :----------------------------------- | :----------- | :----------------------------------------------- | :------- |
+| roleIdOrLabel | `id` or `label` of the Role          | URL          | String                                           | TRUE     |
+
+#### Response parameters
+
+Requested custom [Role](#custom-role-object)
+
+#### Request example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/iam/roles/UserCreator"
+```
+
+#### Response example
+
+```json
+{
+  "id": "cr0Yq6IJxGIr0ouum0g3",
+  "label": "UserCreator",
+  "description": "Create users",
+  "permissions": [
+    "okta.users.create",
+    "okta.users.read",
+    "okta.groups.read",
+    "okta.users.profile.manage"
+  ]
+}
+```
+
+### List roles
+<ApiLifecycle access="beta" />
+<ApiOperation method="get" url="/api/v1/iam/roles" />
+
+Gets a paginated list of custom Roles
+
+#### Response parameters
+
+A paginated list of [custom Roles](#custom-role-object)
+
+#### Request example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/iam/roles"
+```
+
+#### Response example
+
+```json
+{
+  "roles": [
+    {
+      "id": "cr0Yq6IJxGIr0ouum0g3",
+      "label": "UserCreator",
+      "description": "..."
+    },
+    {
+      "id": "cr0Fw7HKcWIroo88m3r1",
+      "label": "GroupMembershipManager",
+      "description": "..."
+    }
+  ],
+  "_links": {
+    "next": {
+      "href": "http://${yourOktaDomain}/api/v1/iam/roles?after=cr0Fw7HKcWIroo88m3r1"
+    }
+  }
+}
+```
+
+### List permissions
+<ApiLifecycle access="beta" />
+
+<ApiOperation method="get" url="/api/v1/iam/roles/${roleIdOrLabel}/permissions" />
+
+Gets the list of permissions included in a custom Role identified by its `id` or `label`
+
+#### Request parameters
+
+| Parameter     | Description                          | Param Type   | DataType                                         | Required |
+| :------------ | :----------------------------------- | :----------- | :----------------------------------------------- | :------- |
+| roleIdOrLabel | `id` or `label` of the Role          | URL          | String                                           | TRUE     |
+
+#### Response parameters
+
+An array of [Permission types](#permission-types), which make the Role identified by `${roleIdOrLabel}` and a link to that Role
```

Comment:
An array of Permission types, which make the Role... > An array of *p*ermission types, which make*s* the Role...